services/src/main/java/org/keycloak/services/resources/admin/RealmLocalizationResource.java

```diff
@@ -191,7 +191,7 @@ public Stream<String> getRealmLocalizationLocales() {
     @Tag(name = KeycloakOpenAPI.Admin.Tags.REALMS_ADMIN)
     @Operation()
     @APIResponses(value = {
-        @APIResponse(responseCode = "200", description = "", content = @Content(schema = @Schema(implementation = Map.class, type = SchemaType.ARRAY))),
+        @APIResponse(responseCode = "200", description = "OK"),
```

Comment:
@ahus1 here you have removed the content, why? In the rest of the endpoints it is always set